### PR TITLE
greeting for kottans

### PR DESCRIPTION
**Pull request by simpleil**
Update README.md with my commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,10 +225,17 @@ I'm new in your community,you all are cool^^
...
 -----------------------------------------------------------------
-
 ### In force we trust!
 ![cat-force](gif/force.gif)
 
+
+## Hi, Kottans!
+Happy to become a part of this great community!
+### Let's the training begin!!! ;)
+
```
Comment (OleksiyRudenko) [inline README.md]: One empty line is sufficient to split content into paragraphs. More empty lines do not expand the space between paragraphs and also make source file redundantly longer.

**Comment by OleksiyRudenko** [comment]
@simpleil Thank you!
Please, add back single trailing empty line at the end of the file.
This will allow others to add new content not touching your additions.
Thank you!

**Comment by OleksiyRudenko** [comment]
@simpleil Great, thanks!
Now we have a conflict with `master` branch to merge into.

Will you fix it?

Please, let me know if you need any help.

**Comment by simpleil** [comment]
I hope, I make all right

**Comment by OleksiyRudenko** [comment]
@simpleil Well done! Merging.

**Pull request merged**